crates/sui-types/src/gas_model/gas_v1.rs

```diff
@@ -0,0 +1,489 @@
+// Copyright (c) 2021, Facebook, Inc. and its affiliates
```

Comment:
Double-checking: this is a carbon copy of old code and does not need to be reviewed, correct?

Reply:
correct, there are minor changes to honor the trait and the code was split obviously but all code is untouched and put here in a single file for exactly that reason